docs/security/measures.md

```diff
@@ -19,132 +19,147 @@ Read about these attributes in detail on the
+easily retrieve the user's cookies by making the user send a request to one of the attacker's IPs.
 
-To mitigate this risk, it's advisable to only use HTTPS connections with valid
-certificates and enforce it with HTTP Strict Transport Security ([HSTS]) so
-that the attacker must also require the certificate to retrieve the cookies.
+This is technically mitigated by the `Secure` attribute set in cookies by Authelia, however it's still advisable to
+only use HTTPS connections with valid certificates and enforce it with HTTP Strict Transport Security ([HSTS]) which
+will prevent domains from serving over HTTP at all as long as the user has visited the domain before. This means even
+if the attacker poisons DNS they are unable to get modern browsers to connect to a compromised host unless they can also
+obtain the certificate.
 
-Note that using [HSTS] has consequences. That's why you should read the blog
-post nginx has written on [HSTS].
+Note that using [HSTS] has consequences. That's why you should read the blog post nginx has written on [HSTS].
 
 ## Protection against username enumeration
 
-Authelia adaptively delays authentication attempts based on the mean (average) of the
-previous 10 successful attempts, and a small random interval to make it even harder to
-determine if the attempt was successful. On start it is assumed that the last 10 attempts
-took 1000ms, this quickly grows or shrinks to the correct value over time regardless of the
-authentication backend.
+Authelia adaptively delays authentication attempts based on the mean (average) of the previous 10 successful attempts
+in addition to a small random interval of time. The result of this delay is that it makes it incredibly difficult to
+determine if the unsuccessful login was the result of a bad password, a bad username, or both. The random interval of 
+time is anything between 0 milliseconds and 85 milliseconds.
 
-The cost of this is low since in the instance of a user not existing it just sleeps to delay
-the login. Lastly the absolute minimum time authentication can take is 250ms. Both of these measures
-also have the added effect of creating an additional delay for all authentication attempts reducing
-the likelihood a password can be brute-forced even if regulation settings are too permissive.
+When Authelia first starts it assumes the last 10 attempts took 1000 milliseconds each. As users login successfully it
+quickly adjusts to the actual time the login attempts take. This process is independent of the login backend you have
+configured.
+
+The cost of this is low since in the instance of a user not existing it just stops processing the request to delay the
+login. Lastly the absolute minimum time authentication can take is 250 milliseconds. Both of these measures also have 
+the added effect of creating an additional delay for all authentication attempts reducing the likelihood a password can 
```

Comment:
Actually, it does not reduce the likelihood of password brute-force if Authelia can handle an infinite amount of requests in parallel so maybe we should simply remove this sentence. (That's why I've just created the ticket around DDoS :) )

Reply:
Ah yeah you're right!